### ISSUE TITLE
xsstracer

### ISSUE THREAD

**Issue by jaikishantulswani** [1N3/Sn1per#142]
getting this issue in every scan 
/usr/bin/xsstracer", line 141, in <module>
    main(sys.argv)
  File "/usr/bin/xsstracer", line 81, in main
    data1 = s.recv(1024)
socket.timeout: timed out

**Comment by jaikishantulswani** [comment]
Hello @1N3 any workout to resolve this issue ?

**Comment by 1N3** [comment]
Hey @jaikishantulswani sorry for the delay here.. it appears that the socket is timing out (socket.timeout: timed out). Have you tried multiple domains/targets?

**Comment by jaikishantulswani** [comment]
@1N3 No, I am running it with single target only ! Any workout to resolve ?

**Comment by 1N3** [comment]
I don't know of any work around. It could mean that the target being scanned is specifically blocking XST or host header injections and results in a timeout. Unfortunately I can't change that behavior but you could try using native curl commands to see if that works (ie. curl -X TRACE https://target.com) and (curl -H "Host: google.com" https://target.com) etc..

**Comment by jaikishantulswani** [comment]
@1N3 , I have also tried it with different targets but result in error is the same, stuck at xsstracer, is that possible to skip that xsstracer event?

**Comment by 1N3** [comment]
Yes, just open the sniper script ```/usr/bin/sniper``` and add a ```#``` before each xsstracer command and that should do the trick.

**Comment by jaikishantulswani** [comment]
Done !
